### PR TITLE
Add documentation for using grafana, prometheus with RisingWave for monitoring of its metrics

### LINKED ISSUE #1635: Document how to deploy RisingWave standalone instance with local disk
Some users prefer to use RisingWave without shared storage (S3)

We won't do much for their request, but here seems to be a low-hanging fruit - standalone cluster + local disk + local embedded compactor

Let's document this and see whether users like it.

### PR DESCRIPTION
**Pull request by kwannoel**
## Info

Close https://github.com/risingwavelabs/risingwave-docs/issues/1635

- **Description**

  - Document how to monitor the metrics of single node deployment with grafana and prometheus.

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -179,6 +179,40 @@ You can view other options with:
 risingwave single --help
 ```
 
+## Monitoring RisingWave Standalone Mode with Grafana and Prometheus
+
+To Monitor your standalone cluster,
+you may wish to integrate metrics monitoring with Grafana and Prometheus.
+
+First install [Grafana]() and [Prometheus]().
```
Comment (WanYixian) [inline docs/get-started.md]:
```suggestion
First install [Grafana](https://grafana.com/docs/grafana/latest/setup-grafana/installation/) and [Prometheus](https://prometheus.io/docs/prometheus/latest/installation/).
```

File: docs/get-started.md [hunk 2]
```diff
@@ -179,6 +179,40 @@ You can view other options with:
...
+
+Next, clone the [RisingWave]() repository, it contains the various configuration files.
```
Comment (WanYixian) [inline docs/get-started.md]:
```suggestion
Next, clone the [RisingWave](https://github.com/risingwavelabs/risingwave) repository, it contains various configuration files.
```

File: docs/get-started.md [hunk 3]
```diff
@@ -179,6 +179,40 @@ You can view other options with:
 risingwave single --help
 ```
 
+## Monitoring RisingWave Standalone Mode with Grafana and Prometheus
+
+To Monitor your standalone cluster,
```
Comment (WanYixian) [inline docs/get-started.md]:
```suggestion
To monitor your standalone cluster,
```

File: docs/get-started.md [hunk 4]
```diff
@@ -179,6 +179,40 @@ You can view other options with:
 risingwave single --help
 ```
 
+## Monitoring RisingWave Standalone Mode with Grafana and Prometheus
```
Comment (WanYixian) [inline docs/get-started.md]:
```suggestion
## Monitoring RisingWave standalone mode with Grafana and Prometheus
```

File: versioned_docs/version-1.8/get-started.md [hunk 5]
```diff
@@ -179,6 +179,41 @@ You can view other options with:
 risingwave single --help
 ```
 
+## Monitoring RisingWave Standalone Mode with Grafana and Prometheus
```
Comment (WanYixian) [inline versioned_docs/version-1.8/get-started.md]:
```suggestion
## Monitoring RisingWave standalone mode with Grafana and Prometheus
```

File: versioned_docs/version-1.8/get-started.md [hunk 6]
```diff
@@ -179,6 +179,41 @@ You can view other options with:
...
+
+To Monitor your standalone cluster,
```
Comment (WanYixian) [inline versioned_docs/version-1.8/get-started.md]:
```suggestion
To monitor your standalone cluster,
```

File: versioned_docs/version-1.8/get-started.md [hunk 7]
```diff
@@ -179,6 +179,41 @@ You can view other options with:
...
+you may wish to integrate metrics monitoring with Grafana and Prometheus.
+
+First install [Grafana]() and [Prometheus]().
```
Comment (WanYixian) [inline versioned_docs/version-1.8/get-started.md]:
```suggestion
First install [Grafana](https://grafana.com/docs/grafana/latest/setup-grafana/installation/) and [Prometheus](https://prometheus.io/docs/prometheus/latest/installation/).
```

File: versioned_docs/version-1.8/get-started.md [hunk 8]
```diff
@@ -179,6 +179,41 @@ You can view other options with:
...
+
+Next, clone the [RisingWave]() repository, it contains the various configuration files.
```
Comment (WanYixian) [inline versioned_docs/version-1.8/get-started.md]:
```suggestion
Next, clone the [RisingWave](https://github.com/risingwavelabs/risingwave) repository, it contains various configuration files.
```

**Review by WanYixian — APPROVED** [review]
Add some links and LGTM now

**Comment by kwannoel** [comment]
> Add some links and LGTM now

Thank you!

**Comment by kwannoel** [comment]
Waiting for https://github.com/risingwavelabs/risingwave/pull/16594 to be reviewed and merged, otherwise the instructions here point to missing files.

**Review by fuyufjh — APPROVED** [review]

**Pull request merged**